roles/engineer.md

```diff
@@ -18,6 +16,21 @@ We expect someone in the role:
 - to be autonomously delivering features of a small nature within 3 months
 - to be autonomously delivering features of a medium nature within 6 months
 
+
+## Academy Engineer role
+
+We need someone:
+
+- to train to be ready to join a delivery team following graduation
+
+We expect someone in the role:
+
+- to be autonomously delivering features of a micro nature before graduation
```

Comment:
Have we got any better language for this? Agree with what you said earlier about it being unclear.